### PR TITLE
#166412712 Fix database fail on travis

### PR DESCRIPTION
**Pull request by encodedBicoding**
### What does this PR do?
Updates `before_script` hook to include the following scripts:
 ``` 
  - psql -c 'create database travis_ci_test;' -U postgres
  - npm run migrate
  - npm run seed

```
The travis was initially testing via the development environment, so the following script was added to make it test via the test environment:
```
env:
 global:
   - export NODE_ENV=test
```
lastly, the `TEST_DATABASE_URL` environment variable on travis was set to:
`postgres://postgres@127.0.0.1:5432/travis_ci_test`
#### Description of Task to be completed?
Update scripts on travis.yml
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166412712 ](https://www.pivotaltracker.com/story/show/166412712)
#### Screenshots (if appropriate)
Screenshot (58)

#### Questions:

### CODE REVIEW

**Review by PelumiAlesh — DISMISSED** [review]

**Review by chuxmykel — DISMISSED** [review]
LGTM

**Review by OvieMudi — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
+    "pretest": "npm run migrate:undo && npm run migrate && npm run seed",
     "test": "cross-env DEBUG=dev jest --forceExit --verbose --maxWorkers=1 --coverage",
-    "coverage": "jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
+    "coverage": "npm run pretest && jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
```
Comment (cvjude) [inline package.json:10]: Why is npm run pretest here

Comment (encodedBicoding) [reply package.json:10]: Because coverage was failing

Comment (encodedBicoding) [reply package.json:10]: It was still seeing the data when it runs, the scripts drops and create for coverage to work with fresh data

Comment (cvjude) [reply package.json:10]: Ok

**Review by ocranbillions — DISMISSED** [review]

**Review by cvjude — DISMISSED** [review]

**Review by cvjude — DISMISSED** [review]

**Review by ocranbillions — DISMISSED** [review]

**Comment by emp-daisy** [comment]
Can you remove the snapshot files and gitignore them?

File: package.json [hunk 2]
```diff
@@ -5,8 +5,13 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
+<<<<<<< HEAD
     "pretest": "cross-env NODE_ENV=test npm run migrate:undo && cross-env NODE_ENV=test npm run migrate && cross-env NODE_ENV=test npm run seed",
     "test": "cross-env DEBUG=dev jest --forceExit --verbose --maxWorkers=1 --coverage",
+=======
+    "pretest": "npm run migrate:undo && npm run migrate && npm run seed",
+    "test": "cross-env DEBUG=dev jest --updateSnapshot --forceExit --verbose --maxWorkers=1 --coverage",
+>>>>>>> 3395de6e5a306be417fd8b3be2bbf028d39d27ec
```
Comment (emp-daisy) [inline package.json]: You didn't handle merge conflict well

Comment (encodedBicoding) [reply package.json]: Oops, I didn't see that file. Will fix that ASAP. Still on my way

**Review by cvjude — DISMISSED** [review]

**Review by Hadeneekeh — APPROVED** [review]

**Pull request merged**